### ISSUE TITLE
Should right axis title rotate the same way as left axis title by default

### ISSUE THREAD

**Issue by kanitw** [vega/vega#3571]
Looking at Vega-Lite dual axis example:

<img width="493" alt="image" src="https://user-images.githubusercontent.com/111269/189539956-7e6beb51-d6f6-4af9-96d2-dcf8a3fc3d44.png">

Readers currently have to tilt their head in two different ways to read the chart.

Alternatively, if the right axis label title gets rotated in the same way, it might be easier to read. 

<img width="489" alt="image" src="https://user-images.githubusercontent.com/111269/189539993-31d3e966-cd05-4eeb-b44f-27b25e3f7ec1.png">

**Comment by domoritz** [comment]
Yes, I think that would be good. It's especially important for dual-axis charts as you point out but wouldn't hurt for charts with only one axis on the right side.

**Comment by domoritz** [comment]
I disagree now with my past self.
